conftest.py

```diff
@@ -31,3 +31,17 @@ def destory_dist():
             pass
 
     request.addfinalizer(destory_dist)
+
+
+@pytest.fixture(scope="function", autouse=True)
+def random_seed():
```

Comment:
Would `fix_random_seed` a better name?

Reply:
This name doesn't matter as it is an autouse fixture.